src/scripts/content/google-classroom.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+togglbutton.render('div[data-without-stream-item-materials]', { observe: true }, elem => {
+  if ($('.toggl-button', elem)) {
+    return;
+  }
+
+  const titleFunc = () => {
+    const title = $('h1', elem);
+    return title ? title.textContent.trim() : '';
+  };
+
+  const link = togglbutton.createTimerLink({
+    className: 'google-classroom',
+    description: titleFunc
+  });
+
+  $('.rec5Nb.QRiHXd', elem).appendChild(link);
```

Comment:
Let's try using the header element as the pivot so we avoid the generated selector (from our experience it'll break very soon). It should look more or less like this:
![image](https://user-images.githubusercontent.com/371426/66089551-42ab9500-e556-11e9-9ec4-b5c00e47bac0.png)

```suggestion
  $('h1', elem).nextSibling.appendChild(link);
```

You'd have to add some styling to the link too, this should be enough:
```
  link.style.margin = '10px 20px';
  link.style.whiteSpace = 'nowrap';
```